spec/std/process_spec.cr

```diff
@@ -81,6 +103,22 @@ describe Process do
     $?.exit_code.should eq(0)
   end
 
+  it "chroot raises when unprivileged" do
+    status, output = build_and_run <<-CODE
+      begin
+        Process.chroot("/usr")
+        puts "FAIL"
+      rescue ex : Errno
+        puts ( ex.errno == Errno::EPERM ) ? "Success" : "FAIL: \#{ex.message||"No Message"}"
```

Comment:
- Please, remove whitespaces after `(` & before `)`
- `||"No Message"` part here is redundant, `#{ex.message}` is enough. ditto below
- Would be a tad cleaner to use non-interpolated heredoc (`<<-'CODE'`) as suggested by @straight-shoota, instead of escaping interpolation

Reply:
We very much need a style guide. @sija You seem to be very interested in compelling a specific style, not throwing shade I also have strong style opinions. Perhaps you can draft a Style Guide that can be discussed and adopted. The space before and after brackets `formats`. I am going to change it this time but I think this is unnecessarily micro management of style.

Reply:
@chris-huxtable we have a style guide right now - `crystal tool format`. This code doesn't format.

I don't care that it doesn't format because it's inside a string, inside a the spec suite, and it doesn't break the build, and honestly who cares, but it seems some people do.